src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/model/EmailContent.java

```diff
@@ -15,6 +18,8 @@
         private String emailAddress;
         private String createdAt;
 
+        private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
I think it would be cleaner to do the objectMapping in the AvroSerializer and keep this pojo as a pojo rather than add any processing into it. That way I think it keeps this class in line with the 'Model' folder it lives in.

Reply:
To add to Marks comment, maybe we should change the createdAt field to contain a date object (LocalDateTime maybe?) and let the serializer deal with mapping that too, as it stands the time needs to be mapped before it is passed in

Reply:
Moved field conversion handling to AvroSerializer.